### PR TITLE
Feature/hints

### PR DESCRIPTION
**Pull request by Janchorizo**
A series of hints have been implemented to provide guidance in the use of the platform. This includes:
- An introductory message that shows when the user has no projects.
- An toggle-able overlay in the project land page.
- Bottom toggle-able panel hints for the TEI stats, dashboard list, dashboard and annotator.

This is a work in progress that will require revisiting the text content, sample document and possibly
adding the option to completely dismiss all hints.

### CODE REVIEW

**Comment by Janchorizo** [comment]
Known issue: Project overlay is not completely responsive. This is to be done when the content is agreed.

File: src/collaborative_platform/apps/help/static/help/tei_sample_document.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <fileDesc>
+      <titleStmt>
+        <title>To make Pease Soop</title>
+      </titleStmt>
+      <publicationStmt>
+        <publisher>
+          <orgName>Des Zentrums für Gastrosophie</orgName>
+          <ptr target="http://gastrosophie.sbg.ac.at/"/>
+        </publisher>
+        <date when="">2015-02-11</date>
```
Comment (virdiignis) [inline src/collaborative_platform/apps/help/static/help/tei_sample_document.xml:13]: I think that filling _when_ parameter here might be a good idea, the sample file should be as correct and abundant as possible.

File: src/collaborative_platform/apps/help/static/help/tei_sample_document.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        <p>Some random date <date xml:id="date-0" name="sam's birthday" ref="#date-0" when="2020-07-04">4th of july 2010</date> for test unlistable entities, and reference to this date <date xml:id="date-1" ref="#date-0">the exact same day</date>. Another important date <date xml:id="date-66" ref="#date-66" when="1791-05-03">3rd of may 1791</date>.</p>
```
Comment (virdiignis) [inline src/collaborative_platform/apps/help/static/help/tei_sample_document.xml:152]: _name_ parameter is not permitted within _date_ tag :(

File: src/collaborative_platform/apps/help/templatetags/no_projects_hint.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from django import template
+
+
+register = template.Library()
+
+@register.inclusion_tag('help/no_projects_hint.html')
+def no_projects_hint():
+    return {}
```
Comment (virdiignis) [inline src/collaborative_platform/apps/help/templatetags/no_projects_hint.py:8]: Please make code pep8-complaint (newline at the end of file in that case)

File: src/collaborative_platform/apps/help/urls.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from . import views
+
+
+app_name = 'projects'
+
```
Comment (virdiignis) [inline src/collaborative_platform/apps/help/urls.py:6]: Not sure why this code resides in _help_ app?

File: src/collaborative_platform/apps/help/views.py [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+from django.utils.decorators import method_decorator
+
+from apps.core.models import Profile
+from apps.files_management.models import Directory
+from apps.views_decorators import objects_exists, user_has_access
+from apps.projects.loggers import ProjectsLogger
+
+from .forms import ContributorForm, ProjectEditForm
+from .models import Project, Contributor
+
```
Comment (virdiignis) [inline src/collaborative_platform/apps/help/views.py:18]: Same for this code. Why is it copied to _help_ app?

**Review by virdiignis — CHANGES_REQUESTED** [review]
I've expressed my doubts in comments to files. Other than these, I see no problem in this PR.

File: src/collaborative_platform/apps/help/views.py (hunk 5 above)
Comment (Janchorizo) [reply src/collaborative_platform/apps/help/views.py:18]: Hum, my mistake. I initially forked from it and left those two bits behind.

File: src/collaborative_platform/apps/help/static/help/tei_sample_document.xml (hunk 2 above)
Comment (Janchorizo) [reply src/collaborative_platform/apps/help/static/help/tei_sample_document.xml:152]: It is now not permitted indeed. This file, however, is taken from the ones provided for testing as a placeholder. 
It is my intention to create a file specifically for this where the content is a brief introduction to the project, platform
and that uses all entities and annotations.

Comment (Janchorizo) [reply src/collaborative_platform/apps/help/static/help/tei_sample_document.xml:152]: I will update this one and, later on, work on this better alternative I have described.

**Pull request merged**